### ISSUE TITLE
Exploring the alteration of gut microbiota and brain function in gender-specific Parkinson's disease based on metagenomic sequencing

### ISSUE THREAD

**Issue by Omabekee** [waldronlab/BugSigDBcuration#271]
Exploring the alteration of gut microbiota and brain function in gender-specific Parkinson's disease based on metagenomic sequencing --Zhang et al. -- Frontiers in Aging Neuroscience
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10370496/

**Comment by keamybams** [comment]
Hello @SvetlanaUP My previous curation is completed and I would love to curate this while waiting for review 

Ruth Bamgbose

Thank you.

**Comment by keamybams** [comment]
Hello @SvetlanaUP , this curation is ready for review https://bugsigdb.org/Study_979. Thanks.

**Comment by Scholarpat** [comment]
Hi @SvetlanaUP,

I've gone through this curation and made some additions. Fig. 2C (which is curatable) was curated initially. I have also included Fig. 2E, which is also curatable/significant. Additionally, I adjusted the group names for clarity. However, the signature is not adopting the Group 1 name. Could this be due to the recent bugsigdb update?

I have documented this challenge under this [issue](https://github.com/waldronlab/BugSigDB/issues/230).

Thank you.
